### DISCUSSION TITLE
Is BudiBase a fit for my app?

### DISCUSSION THREAD

**Discussion by thalperi** [Budibase/budibase#13952] [Help] [answered]
I started a thread on BudiBase's Discord server to explain my app objectives and to ask if BudiBase is a fit for my app. I haven't received any response and I'm wondering if maybe Github is a better way to start a conversation about this:
https://discord.com/channels/733030666647765003/1251049019418087424

**Comment by ConorWebb96** [comment, marked as answer]
Going to close as I believe you've reached out to sales to talk about this usecase.
